qiskit/circuit/library/n_local/evolved_operator_ansatz.py

```diff
@@ -106,6 +106,9 @@ def num_qubits(self) -> int:
         if self.operators is None:
             return 0
 
+        if isinstance(self.operators, list) and len(self.operators) == 0:
+            return 0
+
         if isinstance(self.operators, list) and len(self.operators) > 0:
             return self.operators[0].num_qubits
```

Comment:
We could merge this with the if-clause below:
```suggestion
        if isinstance(self.operators, list):
            if len(self.operators) == 0:
                return 0
            return self.operators[0].num_qubits
```

Reply:
sure

Reply:
Thanks for addressing the other comments! Did you still want to update this?

Reply:
Sorry, I thought that I've done this. Now it's all sorted.